src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -1190,19 +1190,26 @@ Color.white, new Procedure("condition4"),
 			block.enablePitch = !_true;
 			block.customProperties = new ArrayList<>();
 			if (!emptyLists) {
-				block.customProperties.add(new PropertyDataWithValue<>(new PropertyData.LogicType("bool_prop"), _true));
-				block.customProperties.add(
-						new PropertyDataWithValue<>(new PropertyData.LogicType("bool_prop2"), _true));
+				block.customProperties.add(new PropertyDataWithValue<>(

```

Comment:
It looks like those tests only provide custom properties. Shouldn't they provide some vanilla ones too to do full test of the system?

Reply:
I added a known vanilla property and a custom property with vanilla-reserved name for cases when block is not waterloggable. However, tests for dialog are still missing changes from here indeed.

Reply:
You are only testing one tye. Is there a reason for this?

Reply:
We would need to add some more code to get properties and their value bounds from data list.

Reply:
We know the bounds and values, so you can hardcode them here for the test. If they change, we also alter tests.

Same if range changes of some JSpinner, for example, one also adjusts the tests

Reply:
:heavy_check_mark: Fixed